src/main/java/com/solace/spring/cloud/stream/binder/inbound/InboundXMLMessageListener.java

```diff
@@ -19,46 +20,33 @@
 import org.springframework.lang.Nullable;
 import org.springframework.messaging.Message;
 
-import java.util.List;
-import java.util.Optional;
 import java.util.concurrent.atomic.AtomicBoolean;
 import java.util.concurrent.atomic.AtomicInteger;
 import java.util.function.Consumer;
 import java.util.function.Supplier;
-import java.util.stream.Collectors;
 
 @Slf4j
 abstract class InboundXMLMessageListener implements Runnable {
     final FlowReceiverContainer flowReceiverContainer;
     final ConsumerDestination consumerDestination;
     private final ExtendedConsumerProperties<SolaceConsumerProperties> consumerProperties;
     final ThreadLocal<AttributeAccessor> attributesHolder;
-    private final BatchCollector batchCollector;
     private final XMLMessageMapper xmlMessageMapper;
     private final Consumer<Message<?>> messageConsumer;
     private final JCSMPAcknowledgementCallbackFactory ackCallbackFactory;
     @Nullable
     private final SolaceMeterAccessor solaceMeterAccessor;
     private final boolean needHolder;
     private final boolean needAttributes;
+    @Getter
     private final AtomicBoolean stopFlag = new AtomicBoolean(false);
     private final Supplier<Boolean> remoteStopFlag;
+    private final LargeMessageSupport largeMessageSupport = new LargeMessageSupport();
 
-    InboundXMLMessageListener(FlowReceiverContainer flowReceiverContainer,
-                              ConsumerDestination consumerDestination,
-                              ExtendedConsumerProperties<SolaceConsumerProperties> consumerProperties,
-                              @Nullable BatchCollector batchCollector,
-                              Consumer<Message<?>> messageConsumer,
-                              JCSMPAcknowledgementCallbackFactory ackCallbackFactory,
-                              @Nullable SolaceMeterAccessor solaceMeterAccessor,
-                              @Nullable AtomicBoolean remoteStopFlag,
-                              ThreadLocal<AttributeAccessor> attributesHolder,
-                              boolean needHolder,
-                              boolean needAttributes) {
+    InboundXMLMessageListener(FlowReceiverContainer flowReceiverContainer, ConsumerDestination consumerDestination, ExtendedConsumerProperties<SolaceConsumerProperties> consumerProperties, Consumer<Message<?>> messageConsumer, JCSMPAcknowledgementCallbackFactory ackCallbackFactory, @Nullable SolaceMeterAccessor solaceMeterAccessor, @Nullable AtomicBoolean remoteStopFlag, ThreadLocal<AttributeAccessor> attributesHolder, boolean needHolder, boolean needAttributes) {
```

Comment:
@helios57 Was the editorconfig correct here?

Reply:
The .editorconfig was added and not everything reformatted to avoid big changes. But we need to format it.